app/controllers/comments_controller.rb

```diff
@@ -24,6 +24,20 @@ def edit
   # POST /comments
   # POST /comments.json
   def create
+    captcha = params['captcha']
+
+    if captcha.nil?
+      flash[:error] = 'Captcha is required!'
+      return redirect_back(fallback_location: root_path)
+    end
+
```

Comment:
Please also add check if comment content is empty. This way users can store empty comments.